### DISCUSSION TITLE
Will MSC1228 be implemented in this server implementation?

### DISCUSSION THREAD

**Discussion by grahhnt** [matrix-org/dendrite#2487] [General]
**Description:**

Is [MSC1228](https://github.com/matrix-org/matrix-spec-proposals/pull/1228) being considered for this server implementation?

I feel like getting this worked on earlier in development would make it easier to implement. Having this MSC added and implemented would make it easier and more appealing for new communities to move over to using Matrix.

**Comment by spaetz** [comment]
I am not a contributing developer, but given that this is a draft MSC which basically is very incompatible with how all of matrix works (by relying on matrix IDs as identifier), I would not hold my breath. The devs have stated that they target released matrix specs rather than experimental MSCs and given that much core functionality is not yet implemented at all (search, room versions >6), I think users have more immediate pain points.

Given the main dev's involvement, I'd rather think that Portable identities [MSC 2787](https://github.com/matrix-org/matrix-spec-proposals/pull/2787) have a higher chance to be incorporated, but also there I would not bet in the short term.
